code-reviews/recipes/Terraform.md

```diff
@@ -1,18 +1,23 @@
-# Terraform Code Reviews
+## Code Reviews
```

Comment:
For consistency with other recipes change back to `# Terraform Code Reviews`

Reply:
updated based on the comment.